Don't unset user or vendor-specified values in the classpath. This allows us to
include BCFIPS within the classpath at runtime

diff --git a/bin/catalina.sh b/bin/catalina.sh
index e312cf5940..a2f09c1e6c 100755
--- a/bin/catalina.sh
+++ b/bin/catalina.sh
@@ -157,10 +157,6 @@ PRGDIR=`dirname "$PRG"`
 # Copy CATALINA_BASE from CATALINA_HOME if not already set
 [ -z "$CATALINA_BASE" ] && CATALINA_BASE="$CATALINA_HOME"
 
-# Ensure that any user defined CLASSPATH variables are not used on startup,
-# but allow them to be specified in setenv.sh, in rare case when it is needed.
-CLASSPATH=
-
 if [ -r "$CATALINA_BASE/bin/setenv.sh" ]; then
   . "$CATALINA_BASE/bin/setenv.sh"
 elif [ -r "$CATALINA_HOME/bin/setenv.sh" ]; then
